benchmark/opperf/nn_operations/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This module seems less urgently needed than measurement of the actual operators, since the execution is asynchronous.

Reply:
Agreed. Making it all ndarray operators only for this initial work.